documentation/get-started/tutorials/get-started-qfc.en.md

```diff
@@ -180,7 +180,22 @@ A form will ask you for project name, description and local directory. In the lo
 
 There are three ways to share a project with an organization:
 
-**Option 1: Uploading directly to the organization:**
+**Option 1: Directly convert your local project to an Organization QFieldCloud project:**
```

Comment:
Convert or create?

Also above it say "three ways to share", isn't more "three ways to make a project being owned" (needs better English)?

Reply:
Good review